tests/Fixtures/App/var/cache/AppPhpcrDebugProjectContainer.xml.meta

```diff
@@ -0,0 +1 @@
maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/tests/Fixtures/App/Resources/SonataCoreBundle/translations";i:1;b:0;}}i:95;C:55:"Symfony\Component\Config\Resource\FileExistenceResource":159:{a:2:{i:0;s:132:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/tests/Fixtures/App/Resources/SonataBlockBundle/translations";i:1;b:0;}}i:96;C:51:"Symfony\Component\Config\Resource\DirectoryResource":123:{a:2:{i:0;s:99:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/src/Resources/translations";i:1;N;}}i:97;C:55:"Symfony\Component\Config\Resource\FileExistenceResource":172:{a:2:{i:0;s:145:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/tests/Fixtures/App/Resources/SonataDoctrinePHPCRAdminBundle/translations";i:1;b:0;}}i:98;C:55:"Symfony\Component\Config\Resource\FileExistenceResource":152:{a:2:{i:0;s:125:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/tests/Fixtures/App/Resources/TwigBundle/translations";i:1;b:0;}}i:99;C:55:"Symfony\Component\Config\Resource\FileExistenceResource":162:{a:2:{i:0;s:135:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/tests/Fixtures/App/Resources/CmfTreeBrowserBundle/translations";i:1;b:0;}}i:100;C:55:"Symfony\Component\Config\Resource\FileExistenceResource":161:{a:2:{i:0;s:134:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/tests/Fixtures/App/Resources/DoctrinePHPCRBundle/translations";i:1;b:0;}}i:101;C:55:"Symfony\Component\Config\Resource\FileExistenceResource":157:{a:2:{i:0;s:130:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/tests/Fixtures/App/Resources/FrameworkBundle/translations";i:1;b:0;}}i:102;C:55:"Symfony\Component\Config\Resource\FileExistenceResource":156:{a:2:{i:0;s:129:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/tests/Fixtures/App/Resources/SecurityBundle/translations";i:1;b:0;}}i:103;C:55:"Symfony\Component\Config\Resource\FileExistenceResource":155:{a:2:{i:0;s:128:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/tests/Fixtures/App/Resources/MonologBundle/translations";i:1;b:0;}}i:104;C:55:"Symfony\Component\Config\Resource\FileExistenceResource":157:{a:2:{i:0;s:130:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/tests/Fixtures/App/Resources/WebServerBundle/translations";i:1;b:0;}}i:105;C:55:"Symfony\Component\Config\Resource\FileExistenceResource":111:{a:2:{i:0;s:85:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/translations";i:1;b:0;}}i:106;C:55:"Symfony\Component\Config\Resource\FileExistenceResource":141:{a:2:{i:0;s:114:"/home/maximilian/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle/tests/Fixtures/App/Resources/translations";i:1;b:0;}}i:107;C:56:"Symfony\Component\Config\Resource\ClassExistenceResource":90:{a:2:{i:0;s:64:"Symfony\Bundle\WebServerBundle\DependencyInjection\Configuration";i:1;b:0;}}}
```

Comment:
Please add new line

Reply:
This one should be ignored instead. I mean the complete file.